packages/core/core/package.json

```diff
@@ -37,13 +37,13 @@
     "core-js": "3.40.0",
     "http-errors": "2.0.0",
     "http-status-codes": "2.3.0",
+    "lodash": "4.17.21",
```

Comment:
Good catch. It's better to *remove* the lodash dependency

Replace `_.capitalize` with this:

```js
function capitalize(str : string) : string {
  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
}
```

The only other place where `lodash` is used in this package is `packages\core\core\test\errors.spec.ts`.

replace `_.isError` with

```js
function isError(value: any): boolean {
  return (
    typeof value === 'object' &&
    value !== null &&
    (value instanceof Error ||
      (typeof value.name === 'string' && typeof value.message === 'string' && typeof value.stack === 'string'))
  );
}
```